tests/utils/touch/closeCard.js

```diff
@@ -0,0 +1,12 @@
+const closeCard = async (browser, cardSelector) => {
+	const contentInfo = await browser.executeAsync(async (cardSelector, done) => {
```

Comment:
you'll need to click the button, and we may not need to return anything in this function, just wait for it to finish